modules/Makefile.am

```diff
@@ -42,6 +42,7 @@ include modules/snmptrapd-parser/Makefile.am
 include modules/tagsparser/Makefile.am
 include modules/xml/Makefile.am
 include modules/appmodel/Makefile.am
+include modules/openbsd/Makefile.am
```

Comment:
Could you also add to the `SYSLOG_NG_MODULES` list below?

Reply:
added

Reply:
done